packages/venia-concept/src/components/Header/header.css

```diff
@@ -56,3 +56,66 @@
     grid-template-columns: 1fr 1fr;
     justify-items: end;
 }
+
+.searchFallback {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 0.5rem;
+    padding: 0 1rem;
+}
+
+.input {
+    composes: input from '../TextInput/textInput.css';
+    max-width: 24rem;
```

Comment:
This value comes from `searchBar.css` as we want the placeholder to look identical to the `searchBar` once it loads. I think this should be a constant somewhere shared between both css files so we only have to update in one place in the future.